WordPress/Classes/ViewRelated/Blog/NoResultsViewController.swift

```diff
@@ -263,4 +270,21 @@ private extension NoResultsViewController {
         return String(format: buttonIdFormat, string)
     }
 
+    // MARK: - `WPAnimatedBox` resource management
+
+    private func startAnimatingIfNeeded() {
+        guard let animatedBox = accessorySubview as? WPAnimatedBox else { return }
+        animatedBox.animate(afterDelay: 0.1)
+    }
+
+    private func stopAnimatingIfNeeded() {
+        guard
```

Comment:
The formatting of this `guard` statement looks weird. Have we done this anywhere else?

Reply:
Typically we have the first `let` on the same line as the `guard`.

Reply:
I have used this formatting in a few places, but I'm happy to make the change.